tools/chrome-debug-extension/README.md

```diff
@@ -44,4 +44,4 @@ When you load a configuration, the contents of the configuration will be shown o
 
 ## Contributing to the tool
 
-This tool is internally open sourced - if you would like to make a contribution, `INSERT_PROCESS_HERE`
+This tool is internally open sourced - if you would like to make a contribution, please follow the process [here](https://github.com/microsoft/ApplicationInsights-JS/tree/ChromeDebugExtension#contributing)
```

Comment:
The ChromeDebugExtension will be removed once we push this into master, so this should point to the master version